sormas-backend/src/main/java/de/symeda/sormas/backend/sormastosormas/SormasToSormasFacadeEjb.java

```diff
@@ -611,27 +632,29 @@ private void saveNewShareInfo(UserReferenceDto sender, SormasToSormasOptionsDto
 	}
 
 	private void sendEntityToSormas(Object entity, String endpoint, SormasToSormasOptionsDto options) throws SormasToSormasException {
-		ObjectMapper mapper = new ObjectMapper();
-		mapper.setVisibility(PropertyAccessor.ALL, Visibility.NONE);
-		mapper.setVisibility(PropertyAccessor.FIELD, Visibility.ANY);
 
-		String userCredentials = StartupShutdownService.SORMAS_TO_SORMAS_USER_NAME + ":" + "sormas2SORMAS";
+		OrganizationServerAccessData serverAccessData = serverAccessDataService.getServerAccessData()
+			.orElseThrow(() -> new SormasToSormasException(I18nProperties.getString(Strings.errorSormasToSormasServerAccess)));
+		OrganizationServerAccessData targetServerAccessData = getOrganizationServerAccessData(options.getOrganization().getUuid())
+			.orElseThrow(() -> new SormasToSormasException(I18nProperties.getString(Strings.errorSormasToSormasServerAccess)));
 
-		HealthDepartmentServerAccessData target =
-			getHealthDepartmentServerAccessData(options.getHealthDepartment().getId()).filter(ad -> !StringUtils.isEmpty(ad.getUrl()))
-				.orElseThrow(() -> new SormasToSormasException(I18nProperties.getString(Strings.errorSormasToSormasServerAccess)));
+		String userCredentials = StartupShutdownService.SORMAS_TO_SORMAS_USER_NAME + ":" + targetServerAccessData.getRestUserPassword();
 
 		Response response;
 		try {
-			response =
-				sormasToSormasRestClient.post(target.getUrl() + endpoint, new String(Base64.getEncoder().encode(userCredentials.getBytes())), entity);
+			byte[] encryptedEntity = encryptionService.encrypt(objectMapper.writeValueAsBytes(entity), targetServerAccessData.getId());
+			response = sormasToSormasRestClient.post(
+				targetServerAccessData.getHostName(),
+				endpoint,
+				new String(Base64.getEncoder().encode(userCredentials.getBytes())),
+				new SormasToSormasEncryptedDataDto(serverAccessData.getId(), encryptedEntity));
```

Comment:
must explicitly state encoding (twice); should probably be done in encryptionService::encrypt?